config/trainer_config.yaml

```diff
@@ -155,9 +129,27 @@ VisualPyramids:
 
 Tennis:
     normalize: true
-    max_steps: 2e7
+    max_steps: 5.0e7
     learning_rate_schedule: constant
+    batch_size: 1024
+    buffer_size: 10240
     hidden_units: 256
+    time_horizon: 1000
+    self_play:
+        window: 10
```

Comment:
Are these hyperparameter changes for Tennis intentional?

Reply:
Added for consistency